capstone/src/main/java/com/google/step/search/ElasticsearchClient.java

```diff
@@ -68,4 +67,20 @@ public void updateRestaurantHeader(RestaurantHeader restaurantHeader) throws IOE
 
     requestFactory.buildPutRequest(requestUrl, putRequestContent).execute();
   }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public List<RestaurantHeader> queryRestaurantHeaders(String query) {
+    throw new UnsupportedOperationException();
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public List<RestaurantHeader> getRandomRestaurants() {
+    throw new UnsupportedOperationException();
+  }
```

Comment:
I'm a bit confused as to why these are unimplemented? Aren't they implemented in an earlier pull request? If the plan is to take the changes in the other pull request feel free to ignore this.